wp-feature-notifications.php

```diff
@@ -15,52 +15,53 @@
  * @package wp-feature-notifications
  */
 
+namespace WP\Notifications;
+
 use WP\Notifications\REST;
 
-if ( ! defined( 'WP_NOTIFICATION_CENTER_PLUGIN_VERSION' ) ) {
-	define( 'WP_NOTIFICATION_CENTER_PLUGIN_VERSION', '0.0.1' );
+if ( ! defined( 'WP_FEATURE_NOTIFICATION_PLUGIN_VERSION' ) ) {
+	define( 'WP_FEATURE_NOTIFICATION_PLUGIN_VERSION', '0.0.1' );
 }
 
-if ( ! defined( 'WP_NOTIFICATION_CENTER_PLUGIN_DIR' ) ) {
-	define( 'WP_NOTIFICATION_CENTER_PLUGIN_DIR', dirname( __FILE__ ) );
+if ( ! defined( 'WP_FEATURE_NOTIFICATION_PLUGIN_DIR' ) ) {
+	define( 'WP_FEATURE_NOTIFICATION_PLUGIN_DIR', dirname( __FILE__ ) );
 }
 
-if ( ! defined( 'WP_NOTIFICATION_CENTER_PLUGIN_DIR_URL' ) ) {
-	define( 'WP_NOTIFICATION_CENTER_PLUGIN_DIR_URL', plugin_dir_url( __FILE__ ) );
+if ( ! defined( 'WP_FEATURE_NOTIFICATION_PLUGIN_DIR_URL' ) ) {
+	define( 'WP_FEATURE_NOTIFICATION_PLUGIN_DIR_URL', plugin_dir_url( __FILE__ ) );
 }
```

Comment:
@Sephsekla the globals that call functions can't be namespaced `const` variables.

Reply:
Ah of course 🤦

Reply:
I left the `WP_FEATURE_NOTIFICATION_PLUGIN_VERSION` as a defined global rather than const for consistency, do you agree?